source/config/.gitignore

```diff
@@ -0,0 +1 @@
+application.properties
```

Comment:
in other projects we've just added /config to the .gitignore in source :man_shrugging: